neo/NeoSystem.cs

```diff
@@ -34,10 +34,11 @@ public class NeoSystem : IDisposable
         public NeoSystem(Store store)
         {
             this.store = store;
+            Plugin.LoadPlugins(this);
```

Comment:
@jsolman, I think that maybe LoadPlugins runs asynchronously, we can still have the UTXO plugin loading after all this, yes?

Reply:
Let me check, if so I’ll need to change that.

Reply:
`LoadPlugins` already runs synchronously.